### ISSUE TITLE
Ability to specify bitrate in lamejs plugin

### ISSUE THREAD

**Issue by natashache** [collab-project/videojs-record#213]
I'm using the `lame.js` plugin to record audio. The recording defaults to a bitrate of 128 kbps. Is there a way to change this? Bitrate is not listed as one of the recording options. Looking through the files and couldn't find where this is set.

**Comment by thijstriemstra** [comment]
@natashache it's possible to change the sampleRate like this: https://github.com/collab-project/videojs-record/blob/master/src/js/plugins/lamejs-plugin.js#L25

But no option for the bitrate yet, I'll look into it.

**Comment by thijstriemstra** [comment]
@natashache i created pull request #214 that should fix this, can you give it a try?

**Comment by natashache** [comment]
@thijstriemstra Ok. Will try this now.

**Comment by natashache** [comment]
@thijstriemstra This doesn't seem to help. Just recorded something with the updated code. Set the audioBitRate option to 64 kbps, but the recorded file still ends up in 128kbps.

Also I noticed that changing the sample rate doesn't work either. No matter what I specify for `audioSampleRate`, it always records in 44100 Hz. 

Here's the clip I just recorded https://mysoundwise.com/tracks/1519429178361e.mp3
Here's the recorded file info:
![screenshot 2018-02-23 18 52 41](https://user-images.githubusercontent.com/19754530/36622347-c0fe7c2e-18ca-11e8-9948-912e1efcf9bd.png)

**Comment by thijstriemstra** [comment]
Can you try recording something with lamejs directly (so without this plugin) and see if these settings work at all over there?

**Comment by thijstriemstra** [comment]
I tested the plugin just now and with default settings it produced:

```
ffprobe version 3.1.3 Copyright (c) 2007-2016 the FFmpeg developers
  built with Apple LLVM version 8.0.0 (clang-800.0.38)
  configuration: --prefix=/usr/local --enable-gpl --enable-nonfree --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libxvid --disable-doc
  libavutil      55. 28.100 / 55. 28.100
  libavcodec     57. 48.101 / 57. 48.101
  libavformat    57. 41.100 / 57. 41.100
  libavdevice    57.  0.101 / 57.  0.101
  libavfilter     6. 47.100 /  6. 47.100
  libswscale      4.  1.100 /  4.  1.100
  libswresample   2.  1.100 /  2.  1.100
  libpostproc    54.  0.100 / 54.  0.100
[mp3 @ 0x7f9bc9802a00] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from '/foo/bitrate.mp3':
  Duration: 00:00:05.62, start: 0.000000, bitrate: 127 kb/s
    Stream #0:0: Audio: mp3, 44100 Hz, mono, s16p, 128 kb/s
```

With these settings:

```
audioSampleRate: 44100,
audioBitRate: 256
```

it produced:

```
[mp3 @ 0x7fa45a013000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from '/foo/bitrate.mp3':
  Duration: 00:00:03.00, start: 0.000000, bitrate: 255 kb/s
    Stream #0:0: Audio: mp3, 44100 Hz, mono, s16p, 256 kb/s
```

And with these settings:

```
audioSampleRate: 44100,
audioBitRate: 64
```

it produces:

```
[mp3 @ 0x7ff3b5000000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from '/foo/bitrate-64.mp3':
  Duration: 00:00:03.76, start: 0.000000, bitrate: 64 kb/s
    Stream #0:0: Audio: mp3, 44100 Hz, mono, s16p, 64 kb/s
```

I'm not sure what browser/version of ffmpeg you're using but `bitRate` seems to work fine here on OSX/Firefox 58.0.2 and ffprobe version 3.1.3.

**Comment by natashache** [comment]
I'm using chrome. Let me try again.

**Comment by thijstriemstra** [comment]
For the issue with 44100khz; there was a bug (fixed in the PR) and I'm now able to produce 22050khz files for example:

```
[mp3 @ 0x7fc669002a00] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from '/foo/bitrate-22.mp3':
  Duration: 00:00:05.28, start: 0.000000, bitrate: 64 kb/s
    Stream #0:0: Audio: mp3, 22050 Hz, mono, s16p, 64 kb/s
```

**Comment by thijstriemstra** [comment]
Also works fine on Chrome:

```
[mp3 @ 0x7f963a804e00] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from '/foo/chrome2.mp3':
  Duration: 00:00:03.79, start: 0.000000, bitrate: 64 kb/s
    Stream #0:0: Audio: mp3, 22050 Hz, mono, s16p, 64 kb/s
```

Will merge #214.

**Comment by thijstriemstra** [comment]
Released as 2.1.2.
